pkg/brigade/project_test.go

```diff
@@ -43,3 +43,37 @@ func TestProjectSecrets(t *testing.T) {
 		t.Error("Project.Repo.SSHKey should not be exported")
 	}
 }
+
+func TestProjectWorkerConfig(t *testing.T) {
```

Comment:
What is this testing?

Reply:
This is testing that `WorkerConfig` can be serialized to/deserialized from JSON.
But I'm open to suggestion like e.g. merging this test into [`TestProjectSecrets`](https://github.com/Azure/brigade/pull/277/files/c6cfb753a2ed9eb6a9e2e47dec549e14d5a742eb#diff-df2235e76b97ddcdf1c446d032d54964R19)!